src/utils/pages.ts

```diff
@@ -7,4 +7,10 @@ export enum Pages {
   smSystemTransfers = '/sm-system/transfers',
   smSystemImportProducts = '/sm-system/import-products',
   smSystemProductsDocuments = '/sm-system/products-documents',
+  smSystemSuppliersOrders = '/sm-system/suppliers-orders',
```

Comment:
```suggestion
  smSystemSuppliersOrders = '/sm-system/suppliers-orders/orders',
```
Zmieniłbym na to raczej, co ty na to?

Reply:
a bo widzę pod spodem `smSystemOrders = '/sm-system/suppliers-orders/orders` ale to jaka jest między nimi różnica?